rust/viz/config.yaml

```
@@ -1,3 +1,3 @@
 framework:
   website: viz.rs
-  version: 0.9
+  version: 0.10
```

```suggestion
  version: '0.10'
```


Otherwise 0.1 will be printed

---

thanks